gsoc/2020/mentors.md

```diff
@@ -5,4 +5,6 @@ layout: plain
 
 ## Full Mentor List (Name, Email, Org)
```

Comment:
Please rebase your branch with the current master and solve the mentors.md conflict